components/serializer.rst

```diff
@@ -894,8 +894,14 @@ The Serializer component provides several built-in normalizers:
     string (``data:...``) such that files can be embedded into serialized data.
 
 :class:`Symfony\\Component\\Serializer\\Normalizer\\DateIntervalNormalizer`
-    This normalizer converts :phpclass:`DateInterval` objects into strings.
-    By default, it uses the ``P%yY%mM%dDT%hH%iM%sS`` format.
+    This normalizer converts a \BackedEnum objects into strings or integers.
+
+:class:`Symfony\\Component\\Serializer\\Normalizer\\BackedEnumNormalizer`
+    This normalizer converts a into strings.
```

Comment:
Something is missing, a bad copy-paste from the line above `:class:`?

Reply:
@alexislefebvre yes perhaps a too nighty PR 😴 
will have a look at it, and also check [your comment here](https://github.com/symfony/symfony-docs/pull/17101#issuecomment-1203949449)